### PR TITLE
docs(canary): add new relic canary configurations

### PR DESCRIPTION
**Pull request by derekmiller**
Now that https://github.com/spinnaker/halyard/pull/1422 is merged, these docs changes should be good to go.

I'm guessing that https://github.com/spinnaker/spinnaker.github.io/blob/master/reference/halyard/commands.md gets pulled over from the auto-generated output of the build, but let me know if not.

### CODE REVIEW

**Review by kevinawoo — APPROVED** [review]

File: setup/canary/index.md [hunk 1]
```diff
@@ -350,3 +351,62 @@ hal config canary prometheus account get
...
+hal config canary newrelic enable
+```
+
+```
+hal config canary newrelic disable
+```
+
+### Manage or view New Relic account information for canary
+
+You can add, delete, and multiple accounts for New Relic service integrations.
```
Comment (dorbin) [inline setup/canary/index.md]: "multiple"
Looks like this is a copy/paste error; all the providers say this. But I'm not sure what this is supposed to be. "...add, delete, and *edit* accounts..."? Edit multiple accounts?

File: setup/canary/index.md [hunk 2]
```diff
@@ -350,3 +351,62 @@ hal config canary prometheus account get
...
+For details on the parameters for these commands, see the [Halyard reference
+documentation](/reference/halyard/commands/#hal-config-canary)
```
Comment (dorbin) [inline setup/canary/index.md]:
```suggestion
documentation](/reference/halyard/commands/#hal-config-canary).
```

**Review by dorbin — COMMENTED** [review]
@derekmiller thanks for adding this. Just a couple of comments.

File: setup/canary/index.md (hunk 1 above)
Comment (derekmiller) [reply setup/canary/index.md]: Good call, I updated that for each to make more sense.

File: setup/canary/index.md (hunk 2 above)
Comment (derekmiller) [reply setup/canary/index.md]: Made this change throughout as well.

File: setup/canary/index.md [hunk 3]
```diff
@@ -255,9 +256,9 @@ hal config canary google disable
 
 ### Manage or view Google account information for canary
 
-You can add, delete, and multiple accounts for Google service integrations.
+You can add, edit and delete multiple accounts for Google service integrations.
```
Comment (dorbin) [inline setup/canary/index.md]: Nit: can you add back in the comma between edit and delete? 
(That'll be in our style guide, when we add it :) )

**Review by dorbin — APPROVED** [review]

File: setup/canary/index.md (hunk 3 above)
Comment (derekmiller) [reply setup/canary/index.md]: As much as I'd like to defend my stance against the Oxford Comma.. 😁 change made!

**Review by dorbin — APPROVED** [review]

**Pull request merged**